google/async_resumable_media/requests/upload.py

```diff
@@ -38,15 +38,12 @@ class SimpleUpload(_request_helpers.RequestsMixin, _upload.SimpleUpload):
         upload_url (str): The URL where the content will be uploaded.
     """
 
-    def transmit(
+    async def transmit(
         self,
         transport,
         data,
         content_type,
-        timeout=(
-            _request_helpers._DEFAULT_CONNECT_TIMEOUT,
-            _request_helpers._DEFAULT_READ_TIMEOUT,
```

Comment:
Are we no longer able to use a tuple timeout with async? Curious to hear what the reason is for this change.

Reply:
correct. Async has a single timeout.